packages/data-union-solidity/contracts/BinanceAdapter.sol

```diff
@@ -53,11 +54,11 @@ contract BinanceAdapter {
 
     function setBinanceRecipientFromSig(address from, address recipient, bytes memory sig) public {
         UserData storage userdata = binanceRecipient[from];
-        uint nextNonce = userdata.nonce.add(1);
+        uint nextNonce = ++(userdata.nonce);
         require(getSigner(recipient, nextNonce, sig) == from, "bad_signature");
         userdata.nonce = nextNonce;
```

Comment:
is this line needed? you incremented userdata.nonce already

Reply:
👍